### PR TITLE
Update Sentry configuration to include environment

### PR DESCRIPTION
**Pull request by kfrz**
## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

### CODE REVIEW

File: config/initializers/app_info.rb [hunk 1]
```diff
@@ -3,4 +3,5 @@
 module AppInfo
   GIT_REVISION = ENV.fetch('GIT_REVISION', 'MISSING_GIT_REVISION')
   GITHUB_URL   = 'https://github.com/department-of-veterans-affairs/vets-api'
+  VAGOV_ENV    = Settings.vsp_environment
```
Comment (omgitsbillryan) [inline config/initializers/app_info.rb:6]: Is this sentry related or otherwise?

**Review by omgitsbillryan — APPROVED** [review]
LGTM

File: config/initializers/raven.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 Raven.configure do |config|
   config.dsn = Settings.sentry.dsn if Settings.sentry.dsn
+  config.environment = Settings.vsp_environment if Settings.vsp_environment
```
Comment (erluti) [inline config/initializers/raven.rb:10]: Is `~` an acceptable value here?  Lighthouse had some things break because somebody put in a filler value in settings that broke the development and staging deployments that didn't have the right `settings.local.yml` to go with it
